website/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@equalstreetnames/website",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openknowledgebe/equalstreetnames.git"
```

Comment:
Should probably be changed to `git+https://github.com/EqualStreetNames/equalstreetnames.git`